Resources/Catalog/LcgFileCatalogClient.py

```diff
@@ -162,13 +170,11 @@ def returnCode( error, value = '', errMsg = '' ):
 #
 #####################################################
 
-class LcgFileCatalogClient( FileCatalogueBase ):
+class LcgFileCatalogClient( object ):
```

Comment:
Why not inheriting from FileCatalogClientBase? Even if not strictly needed, would be cleaner?